data_structure/trie_modern.cpp

```diff
@@ -0,0 +1,156 @@
+#include <iostream>
```

Comment:
Please add the comment structure at the Beijing of the file. This enables the document generating tool to understand and read the file. refer -> https://github.com/TheAlgorithms/C-Plus-Plus/blob/16fec03ac899cdc847f20e553de7850a83caaa71/search/binary_search.h